packages/tests/test/reya_network/general/General.fork.t.sol

```diff
@@ -55,27 +55,27 @@ contract GeneralForkTest is ReyaForkTest, GeneralForkCheck {
 
         assertLe(block.timestamp - ONE_MINUTE_IN_SECONDS, ethUsdNodeOutput.timestamp);
         assertLe(ethUsdNodeOutput.timestamp, block.timestamp);
-        assertApproxEqAbsDecimal(ethUsdNodeOutput.price, 3500e18, 1000e18, 18);
```

Comment:
shall we update prices instead?

Reply:
I don't think this is future proof. Anytime market gets crazy, we end up changing the prices.